Automattic/themes
# Seedlet: Compress the SASS build and add source maps

#### Changes proposed in this Pull Request:
 - Use `clean-css` to compress the css files that we build, to make them smaller for production
- Add source maps to CSS files so that we can see what files the original rules live in

To test:
- Check that you can see the source file in the inspector when looking at CSS rules.
- Check that RTL ignore rules are preserved.
